runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/options/DataflowWorkerLoggingOptions.java

```diff
@@ -49,6 +89,16 @@
   Level getDefaultWorkerLogLevel();
   void setDefaultWorkerLogLevel(Level level);
 
+  @Description("Controls the log level of messages printed to System.out")
```

Comment:
"Controls the log level given to messages printed to System.out. Depending on the worker log level they may or may not be filtered"

(and possibly mention or Javadoc) how to set the filtering for system out specifically? (same below)